### PR TITLE
Setup with gotbook and remove unneded files

### PR DESCRIPTION
**Pull request by hhio618**
### Description
Add warning to README.md
Add intro to Gitbook doc
### Refs
#268 
[TellorDocs#7](https://github.com/tellor-io/TellorDocs/pull/7)

### CODE REVIEW

**Comment by krasi-georgiev** [comment]
Thanks, can you create a gitbook space from your local branch so we can preview this and future  changes before merging. 

Maybe create a gitbook space linked to your main repo and can just merge this and future PRs there so we can preview latest changes

Can you also add in this PR the .gitbook config file to adjust that the root folder is `./docs`

**Comment by hhio618** [comment]
I made required changes.
Please see Gitbook space [here](https://hhio618.gitbook.io/telliot/v/fix-telliot-docs/)

File: docs/contributing.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following section explains various suggestions and procedures to note during
 
 ### First Steps
 * Familiarizing yourself with our [coding style guidelines.](coding-style-guide.md).
-* Familiarizing yourself with the [Makefile](Makefile) commands, for example `build`,`format`, `test`, `lint`.
+* Familiarizing yourself with the [Makefile](https://github.com/hhio618/telliot/blob/fix-telliot-docs/Makefile) commands, for example `build`,`format`, `test`, `lint`.
```
Comment (krasi-georgiev) [inline docs/contributing.md:33]:
```suggestion
* Familiarizing yourself with the [Makefile](../Makefile) commands, for example, `build`, `format`, `test`, `lint`.
```

**Review by krasi-georgiev — COMMENTED** [review]
Thanks, we are getting there.

Can you also add the changelog as this is also usefull to keep in the logs. 

In a follow up PR would be interested to see how can we avoid maintaining a SUMMARY.md file , I guess naming the files properly with some numbers in front would be enough to organize the pages nicely.

**Comment by hhio618** [comment]
> In a follow up PR would be interested to see how can we avoid maintaining a SUMMARY.md file , I guess naming the files properly with some numbers in front would be enough to organize the pages nicely.

Ok, naming seems to be a good idea to me too. Also, to achieve this we need to update GitHub action in TellorDocs repo too.
I think we can remove SUMMARY.md in this PR then.

**Comment by krasi-georgiev** [comment]
> Ok, naming seems to be a good idea to me too. Also, to achieve this we need to update GitHub action in TellorDocs repo too.
I think we can remove SUMMARY.md in this PR then.

yes lets do that and just set the root folder through the .gitbook config file

**Comment by hhio618** [comment]
> For now lets just set the root folder through the .gitbook file and in a follow up PR will arrange the files. We need to merge few of those and do a bit of refactoring in general.

Ok, I just did that. I believe we need a summary and readme in .gitbook.yaml if we want Telliot docs to be compatible with Gitbook, otherwise we need to have our warning in docs/README.md too.

File: README.md [hunk 2]
```diff
@@ -10,6 +10,15 @@
   </a>
 </p>
 
+
+> ## ⚠️ Note!
```
Comment (krasi-georgiev) [inline README.md:14]: Lets move this in `docs/README.md` so that it is visible when looking at the gitbook.

**Comment by krasi-georgiev** [comment]
> Ok, I just did that. I believe we need a summary and readme in .gitbook.yaml if we want Telliot docs to be compatible with Gitbook, otherwise we need to have our warning in docs/README.md too.

What do you mean by `compatible with gotbook`? But yes please move the warning in `docs/REA...`

**Comment by hhio618** [comment]
> What do you mean by `compatible with gitbook`? But yes please move the warning in `docs/REA...`

done
I actually checked this, there is nothing to worry.

**Comment by krasi-georgiev** [comment]
Oooh I just realised that you can even delete the README.md in the root folder as github will show the one from `docs/README.md`

see here:

> If you put your README file in your repository's root, docs, or hidden .github directory, GitHub will recognize and automatically surface your README to repository visitors.

https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-readmes#about-readmes

File: .gitbook.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+root: ./docs/
+
+​structure:  
+    readme: ../README.md
```
Comment (krasi-georgiev) [inline .gitbook.yaml:4]: I don't think that this is needed anymore.

Comment (krasi-georgiev) [inline .gitbook.yaml:4]:
```suggestion
```
I don't think that this is needed anymore.

**Comment by krasi-georgiev** [comment]
Yep looking good, really glad how simple it is now. Thanks for doing this! 

Now lets see how the sync will work.

**Review by krasi-georgiev — APPROVED** [review]

**Comment by hhio618** [comment]
You're welcome! Thanks for your help!
I think we need two other PRs.
1. first for arranging files inside `docs` folder
2. second updating GitHub action to add these arranged files to table of contents

**Comment by krasi-georgiev** [comment]
yep, I would like to do a bit of refactoring of the text as well, but let's fix and test the auto-sync first. Btw I have sent you a request to chat on google meet, it might be easier and quicker to sync for the last PR.

**Comment by hhio618** [comment]
Ok, sure. can you please ping me on Discord or Telegram?  
handle: @hhio618

**Pull request merged**